clients/web/src/views/body/UsersView.js

```diff
@@ -60,14 +59,27 @@ var UsersView = View.extend({
             parentView: this
         }).on('g:resultClicked', this._gotoUser, this);
 
+        if (getCurrentUser() && getCurrentUser().get('admin')) {
+            const userCountPromise = UserCollection.getTotalCount()
+                .done((count) => {
+                    this.usersCount = count;
+                });
+            promiseArray.push(userCountPromise);
+        }
         this.register = settings.dialog === 'register' && getCurrentUser() &&
                         getCurrentUser().get('admin');
+
+        $.when(...promiseArray)
+            .done(() => {
+                this.render();
+            });
     },
 
     render: function () {
         this.$el.html(UserListTemplate({
             users: this.collection.toArray(),
-            getCurrentUser: getCurrentUser,
+            currentUser: getCurrentUser,
```

Comment:
The issue appears to be that future collection `g:changed` events triggered by fetches initiated by the `SortCollectionWidget` are no longer triggering `UsersView.render` (since we use promises, rather than event binding to trigger the initial render).

I see three possible ways to fix this:
1. Bind collection `g:changed` events to `UsersView.render` at the very end of `UsersView.render`, so additional events will cause additional renders, but the first event will not cause a render.
2. Make `SortCollectionWidget` trigger an event on itself when it's clicked. It may be cleaner to have the `SortCollectionWidget` not actually initiate the `Collection.fetch` at all, to prevent it mutating the state of the collection that it's passed. Alternatively, it could initiate the `fetch`, but pass the `fetch`'s promise as an event parameter.
3. Refactor `UsersView` to render its header and its list independently. This is probably the "best" change, as list updates shouldn't need to re-render the header, but it's also the most invasive, as it will require creating several additional templates and possibly additional views.

Regardless of the option chosen, most of option 2 should be implemented in a separate PR.

@girder/developers Thoughts?